CHANGES.md

```diff
@@ -95,6 +94,10 @@
 
 * Migrated the xcube project setup from `setup.py` to the modern `pyproject.toml` format.
 
+* Support `https` file system using
+  [fsspec.implementations.http.HTTPFileSystem)](https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.implementations.http.HTTPFileSystem),
+  which can be accessed via a new data store called `https`.
```

Comment:
```suggestion
* Added new data store `"https"` that uses 
  [fsspec.implementations.http.HTTPFileSystem)](https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.implementations.http.HTTPFileSystem),
  so that the upcoming xcube STAC data store will be able to access files from URLs.
```